index.html

```diff
@@ -4,31 +4,72 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="no-referrer-when-downgrade">
     <title>Document</title>
-    <script type="text/javascript" async defer src="https://rc.d41.co/sync/"></script>
+    <script type="text/javascript" src="https://polyfill.io/v3/polyfill.min.js?features=URL%2Cfetch"></script>
     <script type="text/javascript" src="https://cdn-0.d41.co/tags/dnb_coretag_v5_rc.min.js"></script>
     <script type="text/javascript">
-        var apiKey = '';
+        const url = new URL(window.location.href);
 
-        setInterval(() => {
-            if (apiKey) {
-                console.log('Firing pixel with API Key: ' + apiKey);
+        const coretag = window.document.createElement("script");
+        coretag.type = "text/javascript";
+        coretag.src = `https://cdn-0.d41.co/tags/dnb_coretag_v5${url.searchParams.has('prod') ? '' : '_rc'}.min.js`
```

Comment:
Prefer not to decide env with query parameter. It's very easy for QA to make mistake. 
Either use a dropdown or keep duplicate copies.

Reply:
07e2912dfb6ebb8455ce2339ef11fe716c0ea418